templates/14_tn_configure_k3s.sh

```diff
@@ -0,0 +1,61 @@
+info "Configure Rancher K3s"
+
+if [ "${USE_GPU_ACCEL}" = true ]; then
+    info "Use nvidia container runtime in containerd"
+    config_file="/var/lib/rancher/k3s/agent/etc/containerd/config.toml.tmpl"
```

Comment:
Adding a config.toml.tmpl is not needed according to K3s docs:
https://docs.k3s.io/advanced#nvidia-container-runtime-support

Also, this causes all PODs not to be scheduled after rebooting the system the first time. It does not recover until /var/lib/rancher/k3s/agent/etc/containerd/config.toml.tmpl is deleted.